app/src/main/java/org/simple/clinic/phone/MaskedPhoneCaller.kt

```diff
@@ -3,7 +3,7 @@ package org.simple.clinic.phone
 import io.reactivex.Completable
 
 /**
- * See [TwilioMaskedPhoneCaller].
+ * See [MaskedPhoneCallerImplementation].
```

Comment:
We should rename this to something more similar to what it actually does, like `OfflineMaskedPhoneCaller` or something.

Reply:
Done